CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to the OpenVINO\* Training Extensions
+
+Welcome to the Training Extensions and thank you for your interest in it! Training Extensions is licensed under the Apache* License, Version 2.0. By contributing to the project, you agree to the license and copyright terms therein and release your contribution under these terms.
+
+If you want to help developing this project, you can do it in several ways:
+
+## Asking questions
+
+First, make sure your question was not asked by anybody else [before](https://github.com/openvinotoolkit/training_extensions/issues?q=is%3Aissue+label%3Aquestion). If you cannot find a related issue, create a new one and add the `question` label.
+
+## Suggesting new features
+
+You can suggest the feature by creating a new issue with `enhancement` label, but first, check if it was suggested [before](https://github.com/openvinotoolkit/training_extensions/issues?q=is%3Aissue+label%3Aenhancement).
+
+## Creating bug reports
+
+If you observe any errors or incorrect work of any parts of this repository, you can create a bug report. First, check if bug report about this problem was not created before [here](https://github.com/openvinotoolkit/training_extensions/issues?q=is%3Aissue+label%3Abug). If you don't find a related bug report, submit your own via creating new issue with `bug` label.
+When creating a new bug report, make sure:
+1. You are on the last version of `develop` branch.
+2. You provide enough information (Python version, OS version, libraries and other environment info) for others to easily reproduce this bug\issue.
+3. You provide necessary and sufficient code snippet to reproduce the bug (if the bug cannot be reproduced in the already existing code).
+4. You describe expected and actual behavior.
+
+## Creating pull requests
+
+If you have an idea how to enhance code of the existing models, you are welcome to create a pull request. You can also add a new model, which does not exist in the repository yet.
+
+### Adding training code of the model
+
+If you want to add training code of your model to this repository, make sure:
+1. Your model code is licensed under permissive license (Apache-2.0, MIT, BSD, etc).
+2. Source framework of the model is PyTorch\*.
+3. You provide the code containing implementation of [OTE SDK Task interfaces](https://github.com/openvinotoolkit/training_extensions/tree/develop/ote_sdk/ote_sdk/usecases/tasks/interfaces):
+   1. [ITrainingTask interface](https://github.com/openvinotoolkit/training_extensions/blob/develop/ote_sdk/ote_sdk/usecases/tasks/interfaces/training_interface.py) - your model must be trainable on some datasets.
```

Comment:
What does I stands for (in `ITrainingTask`, for example)?

Reply:
Hmm, hmm, ..., Interface Maybe?

Reply:
InterfaceTrainingTask interface?

Reply:
This is not abbreviation, this is class name.

Reply:
though ok, I'll remove `interface` word

Reply:
yeah it's for interface